apps/web/.env.example

```
@@ -46,3 +46,8 @@ OPENAI_API_KEY=
 # E2B
 # -----------------------------------------------------------------------------
 E2B_API_KEY=
+
+# -----------------------------------------------------------------------------
+# Feature flags
+# -----------------------------------------------------------------------------
+IS_DEMO="false"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)